setup.cfg

```
@@ -1,35 +1,73 @@
-[flake8]
-max-line-length = 88
-extend-ignore = E203,F403
+[metadata]
+name = halotools
+version = 0.8
+description = Package designed to analyze N-body simulations and constrain models of cosmology and galaxy evolution.
+author = Andrew Hearin
+author_email = ahearin@anl.gov
+license = BSD
+url = http://astropy.org/
+edit_on_github = True
+github_project = astropy/halotools
+
+
+[build_docs]
+source-dir = docs
+build-dir = docs/_build
+all_files = 1
+
 
 [build_sphinx]
 source-dir = docs
 build-dir = docs/_build
 all_files = 1
 
+
 [upload_docs]
 upload-dir = docs/_build/html
 show-response = 1
 
+
 [tool:pytest]
-addopts = --doctest-ignore-import-errors
-minversion = 2.2
+minversion = 6.0
 norecursedirs = build docs/_build
 doctest_plus = enabled
+addopts = --doctest-ignore-import-errors
 markers =
     slow
     installation_test
 
+
+[flake8]
+max-line-length = 88
+extend-ignore = E203,F403
+
+
 [ah_bootstrap]
 auto_use = True
 
-[metadata]
-package_name = halotools
-description = Package designed to analyze N-body simulations and constrain models of cosmology and galaxy evolution.
-long_description = Package designed to analyze N-body simulations and constrain models of cosmology and galaxy evolution.
-author = Andrew Hearin
-author_email = ahearin@anl.gov
-license = BSD
-url = http://astropy.org/
-edit_on_github = True
-github_project = astropy/halotools
+
+[options]
+python_requires = >=3.9
+
+install_requires=
```

No idea if the whitespace is required but might as well be consistent.
```suggestion
install_requires =
```